<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Petri-net Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1944</width>
         <height>1104</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="MatejkoCanvas" name="matejkoCanvas" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1920</width>
            <height>1080</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_project"/>
   <addaction name="actionSave_project"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionToggleSimulationMode"/>
  </widget>
  <widget class="QToolBar" name="simulationToolBar">
   <property name="windowTitle">
    <string>Simulation toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExecuteTransition"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="netAnalysisToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Net analysis toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCoverabilityGraph"/>
   <addaction name="actionReachabilityGraph"/>
   <addaction name="actionLiveness"/>
   <addaction name="actionConservation"/>
   <addaction name="actionBoundedness"/>
   <addaction name="actionIncidence"/>
   <addaction name="actionReversability"/>
  </widget>
  <action name="actionOpen_project">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing project</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current project</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new project</string>
   </property>
  </action>
  <action name="actionExecuteTransition">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Fire Transition</string>
   </property>
   <property name="toolTip">
    <string>Fire the selected transition</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionToggleSimulationMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="system-run">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Simulation mode</string>
   </property>
   <property name="toolTip">
    <string>Turn the simulation mode on/off</string>
   </property>
  </action>
  <action name="actionReachabilityGraph">
   <property name="text">
    <string>Reachability Graph</string>
   </property>
   <property name="toolTip">
    <string>Generate the reachability graph</string>
   </property>
  </action>
  <action name="actionCoverabilityGraph">
   <property name="text">
    <string>Coverability Graph</string>
   </property>
   <property name="toolTip">
    <string>Generate the coverability graph</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save the project as a new file</string>
   </property>
  </action>
  <action name="actionLiveness">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Liveness</string>
   </property>
   <property name="toolTip">
    <string>Generate the liveness report</string>
   </property>
  </action>
  <action name="actionConservation">
   <property name="text">
    <string>Conservation</string>
   </property>
   <property name="toolTip">
    <string>Generate the conservation report</string>
   </property>
  </action>
  <action name="actionBoundedness">
   <property name="text">
    <string>Boundedness</string>
   </property>
   <property name="toolTip">
    <string>Generate the boundedness report</string>
   </property>
  </action>
  <action name="actionIncidence">
   <property name="text">
    <string>Incidence</string>
   </property>
   <property name="toolTip">
    <string>Generate the incidence matrix</string>
   </property>
  </action>
  <action name="actionReversability">
   <property name="text">
    <string>Reversability</string>
   </property>
   <property name="toolTip">
    <string>Generate the reversability report</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatejkoCanvas</class>
   <extends>QWidget</extends>
   <header>matejkocanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
